__tests__/data/hs_input.json

```diff
@@ -572,7 +572,7 @@
           {
             "identifierType": "email",
             "id": "testhubspot2@email.com",
-            "type": "lead"
+            "type": "HS-lead"
```

Comment:
Let's add a new test case for this just to keep the test cases of existing functionality intact.

Reply:
These tests are related to rETL sources if I don't change this will be failing. Here mappedToDestination ia true.